readme.md

```diff
@@ -43,14 +48,55 @@ npm install remark-language-server
 
 Usage of this package depends on your editor integration.
 Because this is based on
-[`vscode-languageserver-node`][vscode-languageserver-node], the same transports
+[`unified-languageserver-node`][unified-languageserver-node], the same features
 are supported.
 
-Practical examples are coming soon.
-
 `remark-language-server` uses the same configuration files as
 [`remark-cli`][remark-cli].
 
+## Examples
+
+### Emacs
+
+Use [`lsp-mode`][lsp-mode] to use the remark language server with Emacs.
+
+### Neovim
+
+The remark language server is part of [`nvim-lspconfig`][nvim-lspconfig], a
+collection of common configurations for Neovim’s built-in language server
+client.
+This means you can add remark language server in one line:
+
+```lua
+require'lspconfig'.remark_ls.setup{}
+```
+
+Detailed installation instructions can be found [here](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#remark_ls).
+
+### vim-lsp
+
+The remark language server can be used with [`vim-lsp`][vim-lsp].
+You can configure it to be started when a markdown file is opened.
+
+```viml
+if (executable('remark-language-server'))
+  au User lsp_setup call lsp#register_server({
+  \ 'name': 'remark',
+  \ 'cmd': {server_info->['remark-language-server', '--stdio']},
+  \ 'allowlist': ['markdown'],
+  \ })
+endif
+```
+
+It has also been registered with [`vim-lsp-settings`][vim-lsp-settings].
+If you use this, you will be prompted to use the remark language server when a
+markdown file is opened.
```

Comment:
```suggestion
Otherwise, if you use [`vim-lsp-settings`][vim-lsp-settings], you will be asked
to use the remark language server automatically when a markdown file is opened.
```

Maybe like this? Less use of “this” and weaker language such as “has also been”